data/remnant missions.txt

```diff
@@ -495,3 +560,713 @@ mission "Remnant: Technology Available"
+	on complete
+		"reputation:  Drak " += 1
+		conversation
+			`Back on Aventine, you share the recording of the void sprite's behavior with Plume. "These Sprites demonstrate considerably more communication than we originally thought. And obviously they must have communicated what you did on Nasqueron. Interesting..." Plume's excited staccato exclamations trail off as he begins to make notes on a datapad he has on hand.`
+
+
+
+mission "Remnant: Learn Sign 1"
+	name "Learn Remnant Sign"
+	description `Visit each of the Remnant worlds to find ways to learn their sign language.`
+	minor
+	source "Caelian"
+	stopover "Aventine"
+	destination "Viminal"
+	to offer
+		has "Remnant: Technology Available: offered"
+		random < 50
+	on offer
+		conversation
+			`As you make your way through the spaceport, you become increasingly aware of how quiet it is. Looking around, you note that it doesn't seem any less busy than usual, just that only a handful of people are paying attention to you. As a result, they have reverted to their normal gestures instead of singing. It feels oddly isolating, and eerie, to realize that there are hundreds of conversations going on around you, yet you can't understand a single word of it.`
+			choice
+				`	(Ask someone to teach you.)`
+				`	(Think about asking later.)`
+					defer
+				`	(Ignore it.)`
+					decline
+					
+			`	Looking around the spaceport, you eventually find someone at what looks remarkably like an information desk.`
+			choice
+				`	"I would like to learn how to talk using signs."`
+					goto words
+				`	(Sing a short verse about not understanding people because your hands are still.)`
+					goto song
+			label words
+			`	She looks at you with a confused expression, her hands making a few quick gestures. After a minute she points at the hand scanner embedded in the desk.`
+				goto scan
+			label song
+			`	She looks surprised, but then recovers and sings a verse about how hands can learn but must be cautious, and gestures at the hand scanner embedded in the desk.`
+			label scan
+			`	As you place your hand on the scanner you feel a quick tingle as it scans your hand, then a pinprick as it extracts a drop of blood. The lady behind the desk looks at her screen. She nods, taps on the screen a few times, then retrieves a datapad from beneath the desk.`
+			`	She hands the pad to you, and sings a brief verse about perseverance and hope.`
+			`	Back on your ship you flip the pad on. The datapad appears to be loaded with a number of historic musicals and operas where the meaning of the lyrics has been replicated with gestures as subtitles. It is interspersed with what appear to be Remnant children's videos slowly explaining gestures and pausing to give you the opportunity to practice.`
+			`	After a few hours a note pops up on the screen, informing you that additional lessons are available from the director's desk on Aventine.`
```

Comment:
It would be the same reason why teachers don't give their students an entire semester's worth of work all at once: If they only give chapters at a time, they can direct the learning and keep track of how their student is progressing. Just because the Remnant aren't handing out grades, doesn't mean they aren't keeping notes.

Reply:
And maybe they want to keep notes on a competent player who can learn their language without being babied along lesson by lesson. :) 
The “learning process” is simply waiting as the player in universe “learns” from the data they receive, so I don’t see an issue with condensing these missions.

Reply:
On the other hand, simply the fact that we *can* condense it down to one doesn't mean we should. The point is to have missions that tell a story surrounding the player, not get it over with as fast as possible. I don't think that speeding it up makes the story any better, or gives the player any additional enjoyment, and does reduce their involvement in the story; which I'd argue is a detriment to their experience.

Reply:
>The point is to have missions that tell a story surrounding the player, not get it over with as fast as possible. I don't think that speeding it up makes the story any better, or gives the player any additional enjoyment, and does reduce their involvement in the story; which I'd argue is a detriment to their experience.

I think that that would be a good argument if there was something actually interactive about these missions or something important that the player as a real life person would need to know, but these missions are simply the player's character learning Remnant sign language while the player themself learns nothing. Given that disconnect, I'm not too fond of having the player have to sit through their character learning.

Reply:
That is a good point. And if learning sign was the only thing this mission did, that'd probably be good justification for removing it. That being said, there's also the secondary element that is intentionally portrayed as a minor part: the blood test. As the director explains at the end of the LS2, part of the point of sending the player around to each planet like that was to have their identity and status as a non-Alpha confirmed independently on each world, thus allowing them to freely work everywhere. 

Given how that ties in with the core identity of the Remnant (and demonstrates their concern even all these years later), I still think it is worth keeping. It also demonstrates that the Remnant aren't above using slightly duplicitous means to repeatedly ensure the security of their worlds.